packages/server/database/migrations/20190904182024-saml.js

```diff
@@ -0,0 +1,20 @@
+exports.up = async (r) => {
+  try {
+    await r.tableCreate('SAML')
```

Comment:
Ah interesting, you decided to put it on its own table. Wasn't sure if it should go here, or on an org. Given 1 IdP can have many orgs, this makes sense